tests/unit_tests/test_Model_Binance.py

```diff
@@ -0,0 +1,274 @@
+    df = exchange.getAccount()
+    assert type(df) is pandas.core.frame.DataFrame
+
+    actual = df.columns.to_list()
+    expected = ['currency', 'balance', 'hold', 'available']
+    assert len(actual) == len(expected)
+    assert all([a == b for a, b in zip(actual, expected)])
+
+
+def test_get_fees_with_market(mocker):
+    client_response = {'success': True, 'tradeFee': [{'maker': 0.001, 'symbol': 'CHZUSDT', 'taker': 0.001}]}
+    with open(_get_config_file()) as config_file:
+        config = json.load(config_file)
+
+        api_key = ''
+        api_secret = ''
+        api_url = ''
+        if 'api_key' in config and 'api_secret' in config and 'api_pass' in config and 'api_url' in config:
+            api_key = config['api_key']
+            api_secret = config['api_secret']
+            api_url = config['api_url']
+            AuthAPI(api_key, api_secret, api_url)
+        elif 'api_key' in config['binance'] and 'api_secret' in config['binance'] and 'api_url' in config['binance']:
+            api_key = config['binance']['api_key']
+            api_secret = config['binance']['api_secret']
+            api_url = config['binance']['api_url']
+            AuthAPI(api_key, api_secret, api_url)
+
+    exchange = AuthAPI(api_key, api_secret, api_url)
+    assert type(exchange) is AuthAPI
+
+    mocker.patch("models.Binance.Client.get_trade_fee", return_value=client_response)
+    df = exchange.getFees(MOCK_MARKET)
+    assert type(df) is pandas.core.frame.DataFrame
+
+    assert len(df) == 1
+
+    actual = df.columns.to_list()
+    expected = ['maker_fee_rate', 'taker_fee_rate', 'usd_volume', 'market']
+    assert len(actual) == len(expected)
+    assert all([a == b for a, b in zip(actual, expected)])
+
+
+def test_get_taker_fee_with_market(mocker):
+    client_response = {'success': True, 'tradeFee': [{'maker': 0.001, 'symbol': 'CHZUSDT', 'taker': 0.001}]}
+    with open(_get_config_file()) as config_file:
+        config = json.load(config_file)
+
+        api_key = ''
+        api_secret = ''
+        api_url = ''
+        if 'api_key' in config and 'api_secret' in config and 'api_pass' in config and 'api_url' in config:
+            api_key = config['api_key']
+            api_secret = config['api_secret']
+            api_url = config['api_url']
+            AuthAPI(api_key, api_secret, api_url)
+        elif 'api_key' in config['binance'] and 'api_secret' in config['binance'] and 'api_url' in config['binance']:
+            api_key = config['binance']['api_key']
+            api_secret = config['binance']['api_secret']
+            api_url = config['binance']['api_url']
+            AuthAPI(api_key, api_secret, api_url)
+
+    exchange = AuthAPI(api_key, api_secret, api_url)
+    assert type(exchange) is AuthAPI
+
+    mocker.patch("models.Binance.Client.get_trade_fee", return_value=client_response)
+    fee = exchange.getTakerFee(MOCK_MARKET)
+    assert type(fee) is float
+    assert fee == 0.001
```

Comment:
Good thing is that now we can assert the exact value we expect. In the original tests, it is `fee > 0`, so if the `exchange.getTakerFee` returns anything different than `0.001`, we know that the change is breaking something.

Reply:
Sounds great.